### PR TITLE
Hide the searchbar placeholder text when it contains text

### PR DESCRIPTION
**Pull request by bjgordon**
Currently the search bar placeholder label has a bug where it is always overlaid on top of any text that is in the search bar:

image

(We are using a label instead of the placeholder attribute for wider IE support)

This PR moves the search bar text in front of the placeholder if it is not empty.

The code that does this on dta.gov.au relied on jquery, so I've rewritten it for the content guide in vanilla js. Once this is working in the content guide I'll likely use it on dta.gov.au. 

I've tested it works in IE 8.

### CODE REVIEW

**Comment by bjgordon** [comment]
@joahua has suggested a way of doing this just in css, will try that

**Comment by bjgordon** [comment]
I wasnt able to get a css only solution, so am reopening this

File: _assets/js/search.js.liquid [hunk 1]
```diff
@@ -7,19 +7,6 @@
 //= require 'vendor/jquery.lunr.search'
 
 $(document).ready(function(){
-    $('.search-input').focus(function() {
```
Comment (simonschwartz) [inline _assets/js/search.js.liquid]: Have you rewritten this somewhere else? This is the function that loads the search index

Comment (bjgordon) [inline _assets/js/search.js.liquid]: No I didnt write it somewhere else. How necessary is it?

**Comment by bjgordon** [comment]
@simonschwartz 
> Why is it an issue not having placeholder text on IE8?

The jquery placeholder way worked in IE 8 so I thought I should try to keep it working...

File: _assets/js/search.js.liquid (hunk 1 above)
Comment (bjgordon) [inline _assets/js/search.js.liquid]: @simonschwartz I've added the prefetching now

**Comment by bjgordon** [comment]
I think that's your issues addressed @simonschwartz. To simplify things I've removed IE 8 support, such that the placeholder text is just shown above the searchbar.

**Comment by dominikwilkowski** [comment]
I'm merging this while noting that labels should not be used this way. This should be looked into again once we are live :)

**Pull request merged**